dist/gmusic.js

```diff
@@ -239,6 +245,87 @@ proto.playback = {
   }
 };
 
+// Create a queue API
+proto.queue = {
+  _fireEvent: function (el, etype) {
+    if (el.fireEvent) {
+      el.fireEvent('on' + etype);
+    } else {
+      var evObj = document.createEvent('Events');
+      evObj.initEvent(etype, true, false);
+      el.dispatchEvent(evObj);
```

Comment:
We are being inconsistent with `e` vs `ev` vs `evt`. We use `evt` elsewhere in the file so please stick with that convention:

```
function (el, evtType) {
  var evt = document.createEvent('Events');
  evt.initEvent(evtType, true, false);
  el.dispatchEvent(evt);
}
```